robotoff/ml/category/prediction_from_image/predictor.py

```diff
@@ -0,0 +1,45 @@
+import joblib
+import numpy as np
+from robotoff.ml.category.prediction_from_image.helpers import list_categories
+from robotoff.ml.category.prediction_from_image.cleaner import Cleaner
+#from robotoff.ml.category.prediction_from_image.ocr import get_data_from_ocr, get_data_from_url
+
+
+class Predictor():
+
+    model = None
+
+    def __init__(self, text):
+        self.text = text
+
+
+    def load_model(self):
```

Comment:
It would be good practice to always put the type annotations. I saw that you've done it in some cases but not all of them. 

For example here it would be :
```py
def load_model(self) -> None:
     (...)
```